hnn_core/tests/assets/test_default.csv

```diff
@@ -0,0 +1,202 @@
+# times,agg,L2,L5
+0.000000, 0.004901, -0.000004, 0.004905
```

Comment:
I suppose this is ok since the file is so small, we generally try to avoid having data files in the commit history, see:
https://github.com/jonescompneurolab/hnn-core/blob/0346dbf5d69a190491d83431e6d4209abc2b61ae/hnn_core/tests/test_parallel_backends.py#L199-L208

@jasmainak @gtdang the file size is just 8kb so I don't think there's much concern. Do you want to leave the test as is or move this to a data branch?

Reply:
My preference would to have small tests files included with the dev install/clone. This allows for offline development and also removes the dependency of an external api.